src/main/java/com/easypost/model/CarrierAccount.java

```diff
@@ -58,11 +58,19 @@ public CarrierAccount update(Map<String, Object> params, String apiKey) throws E
 
 		// delete
 		public void delete() throws EasyPostException {
-			this.delete(null);
+			this.delete(null, null);
 		}
-		public void delete(String apiKey) throws EasyPostException {
+		public void delete(String apiKey, String placeholder) throws EasyPostException {
```

Comment:
hmm yeah I see this issue now; since we want to have both static and instance methods here there's no good way to do this. Java obviously doesn't allow us to have static and instance methods with the same signature, and I don't really like this placeholder solution (since it would require users to pass in empty strings everywhere). I personally think the thing to do would be to make the static methods called something different:  either `destroy` or `deleteById` seem workable to me.